spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/sql/CommandsHarvester.scala

```diff
@@ -458,6 +457,9 @@ object CommandsHarvester extends AtlasEntityUtils with Logging {
     private val RELATION_PROVIDER_CLASS_NAME =
       "org.apache.spark.sql.execution.datasources.hbase.DefaultSource"
 
+    private val JDBC_PROVIDER_CLASS_NAME =
```

Comment:
I'm having some difficulty in understanding what would be required for this comment. Can you link to an example where this has been done?

Reply:
I meant you can add a new object `JDBCEntities` and put your changes there, instead of adding JDBC related code to `HBaseEntities`. (So SHCEntities, HWCEntities, KafkaEntities are all of examples.) 

After then, you can add JDBCEntities into the pattern below: 

```
      case SHCEntities(shcEntities) => Seq(shcEntities)
      case HWCEntities(hwcEntities) => Seq(hwcEntities)
      case KafkaEntities(kafkaEntities) => kafkaEntities
      case JDBCEntities(jdbcEntities) => Seq(jdbcEntities)
```